src/neynar-api/v2/openapi-farcaster/apis/user-api.ts

```diff
@@ -1033,12 +1126,13 @@ export class UserApi extends BaseAPI {
      * @summary Fetches users based on Eth or Sol addresses
      * @param {string} apiKey API key required for authentication.
      * @param {string} addresses Comma separated list of Ethereum addresses, up to 350 at a time
+     * @param {'custody_address' | 'verified_address' | 'all'} [addressTypes] Customize which address types the request should search for. Default is all. Select multiple types by using a comma-separated list of valid types in the enum
      * @param {*} [options] Override http request option.
      * @throws {RequiredError}
      * @memberof UserApi
      */
-    public userBulkByAddress(apiKey: string, addresses: string, options?: AxiosRequestConfig) {
-        return UserApiFp(this.configuration).userBulkByAddress(apiKey, addresses, options).then((request) => request(this.axios, this.basePath));
+    public userBulkByAddress(apiKey: string, addresses: string, addressTypes?: 'custody_address' | 'verified_address' | 'all', options?: AxiosRequestConfig) {
```

Comment:
Need to update wrapper method `fetchBulkUsersByEthereumAddress` in `src/neynar-api/v2/client.ts` and `src/neynar-api/neynar-api-client.ts`

Also the examples in JSDoc in those wrapper methods